.github/workflows/pr-review-notifications.yml

```diff
@@ -11,11 +11,13 @@ on:
 
 jobs:
   pr-reviewed:
+    name: Pull Request Review Submitted
     runs-on: ubuntu-latest
     steps:
       - name: Checkout actions repo # required so we can reference the actions locally
         uses: actions/checkout@v4
         with:
+          ref: david/improve-pr-notifications
```

Comment:
Probably an accidental leftover? :) You can also add `${{ inputs.ref }}` so that downstream clients can indicate which specific version they want.

Reply:
Yes! This can be removed once merged. I was using it for testing.